### PR TITLE
The FAQ had been updated to reflect the latest changes

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -111,15 +111,9 @@ After that, your account is deleted.
...
-No. The act of 'following' a hashtags is an interesting technology, but presents a few issues.
-
-1. Posts would have to be copied to all sites on the network that are "listening" to that hashtag. This would increase the storage demands to the detriment of small sites. It would make the use of shared hosting practically impossible.
-
-2. Making spam easy (tag spam is a serious issue on Twitter for instance)
-
-3. It creates a natural bias towards large sites which hold more tagged content - if your network uses tagging instead of other conversation federation mechanisms such as groups/forums.
-
-Instead, we offer other mechanisms for wide-area conversations while retaining a 'level playing ground' for both large and small sites, such as forums and community pages and shared tags.
+Yes. Simply add the hash tag to your saved searches.
```
Comment (tobiasd) [inline doc/FAQ.md]: Please add a line break after the `Yes.`.

File: doc/de/FAQ.md [hunk 2]
```diff
@@ -122,19 +122,10 @@ Dieses Vorgehen setzt voraus, dass Dein Profil für 24 Stunden weiterhin "teilwe
...
-
-1.) Alle Beiträge, die diesen tag nutzen, müssten zu allen Seiten im Netzwerk kopiert werden. Das erhöht den Speicherbedarf und beeinträchtigt kleine Seiten. Die Nutzung von geteilten Hosting-Angeboten (Shared Hosting) wäre praktisch unmöglich. 
-
-2.) Die Verbreitung von Spam wäre vereinfacht (tag-Spam ist z.B. bei Twitter ein schwerwiegendes Problem)
-
-3.) Der wichtigste Grund der gegen diese Technik spricht ist, dass sie eine natürliche Ausrichtung auf größere Seiten mit mehr getaggten Inhalten zur Folge hat. Dies kann z.B. aufkommen, wenn Dein Netzwerk tags anstelle von anderen Kommunikationsmitteln wie Gruppen oder Foren nutzt. 
-
-Stattdessen bieten wir andere Mechanismen, um globale Unterhaltungen zu erreichen, dabei aber eine angemesse Basis für kleine und große Seiten zu bieten. 
-Hierzu gehören Foren, Gruppen und geteilte tags. 
+Ja. Füge die Tags zu Deinen gespeicherten Suchen hinzu, sie werden automatisch auf der Netzwerk-Seite auftauchen.
```
Comment (tobiasd) [inline doc/de/FAQ.md]: Here as well after the `Ja.`

**Comment by MrPetovan** [comment]
Oops.

**Comment by annando** [comment]
Double oops :-)

**Comment by annando** [comment]
https://github.com/friendica/friendica/pull/4642

**Pull request merged**